scrapers/koji.py

```diff
@@ -160,6 +238,25 @@ def update_neo4j(self, builds):
             user = User.get_or_create(dict(username=username))[0]
             build.conditional_connect(build.owner, user)
 
+            if build.__label__ == ModuleKojiBuild.__label__:
```

Comment:
Why isn't this part of the `elif self.is_module_build(build_dict):` block?

Reply:
Because I wanted to keep that block explicit for checking if the build is created erroneously, just like ContainerKojiBuild. Also, since the change was relating to KojiTag's I wanted to keep it in the same section where other KojiTag's are handled.